src/hw9/ArrayDataException.java

```diff
@@ -1,6 +1,6 @@
 package hw9;
 
-public class ArrayDataException extends Throwable{
-    public ArrayDataException ( String s ){
+public class ArrayDataException extends Throwable {
+    public ArrayDataException (String s){
```

Comment:
Please apply the correct formatting here. You have `public ArrayDataException (String s){`, but must be `public ArrayDataException(String s) {`.
Please note that next time I'll lower the mark for that without any additional warnings as I've been telling you about this many times already.